src/components/orders/OrderPriceDisplay/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { faExchangeAlt } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import BigNumber from 'bignumber.js'
+
+import { calculatePrice, formatSmart, invertPrice, safeTokenName, TokenErc20 } from '@gnosis.pm/dex-js'
+
+const Wrapper = styled.span`
+  display: flex;
+  align-items: center;
+  font-size: ${({ theme }): string => theme.fontSizeNormal};
+`
+
+const Icon = styled(FontAwesomeIcon)`
+  background: ${({ theme }): string => theme.textSecondary2}33; /* 33==20% transparency in hex */
+  border-radius: 1rem;
+  width: 2rem !important; /* FontAwesome sets it to 1em with higher specificity */
+  height: 2rem;
+  padding: 0.4rem;
+  margin-left: 0.5rem;
+`
+
+export type Props = {
+  buyAmount: string | BigNumber
+  buyToken: TokenErc20
+  sellAmount: string | BigNumber
+  sellToken: TokenErc20
+  isPriceInverted?: boolean
+  withoutInvertButton?: boolean
```

Comment:
I would avoid "negation" names. They can make code harder to read, specially when we start to have also complex boolean expressions

But even with a simple true/false value is harder to read:
```bash
withoutInvertButton={false}

// As opposed to
showInvertButton={true}

// ..or
invertButtonVisible={true}
```